qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -367,10 +393,17 @@ def _assemble(self, circuits, parameter_binds=None, **run_options):
     def _assemble_noise_model(self, circuits, **run_options):
         """Move quantum error instructions from circuits to noise model"""
         if isinstance(circuits, (QasmQobj, PulseQobj)):
-            return circuits, run_options
+            optypes = [set() for _ in range(len(circuits.experiments))]
```

Comment:
Unimportant, but since you just need to match the length, it could just be `[set() for _ in circuits.experiments]`.